test/node/src/insert.test.ts

```diff
@@ -317,7 +317,7 @@ for (const dialect of DIALECTS) {
       }
     })
 
-    if (dialect === 'mysql') {
+    if (dialect === 'mysql' || dialect == 'sqlite') {
       it('should insert one row and ignore conflicts using insert ignore', async () => {
```

Comment:
Do I need to add tests for other conflict resolution algorithms too?

Reply:
You can add a unit test for each by looping over a list of `orX` method names.